sklearn/cluster/_spectral.py

```diff
@@ -466,6 +471,7 @@ class SpectralClustering(ClusterMixin, BaseEstimator):
       Stella X. Yu, Jianbo Shi
       https://www1.icsi.berkeley.edu/~stellayu/publication/doc/2003kwayICCV.pdf
     """
+
     def __init__(self, n_clusters=8, *, eigen_solver=None, n_components=None,
```

Comment:
In general it is a good idea not to add changes unrelated with the pull request: this is confusing when years after we check the commit history.

Reply:
Thank you for pointing out, and sorry about this.
This is made by formatter, so I'll disable and modify this.